src/main/java/ur/inf/lab2/pz/servicemanmanagement/domain/validator/FXFormValidator.java

```diff
@@ -18,4 +20,43 @@ public boolean isClean() {
         return !hasErrors;
     }
 
+    public void validateEmptyOrWhitespaceOnly(String text, Text errorLabel) {
+        if (StringUtils.isEmptyOrWhitespaceOnly(text)) {
+            errorLabel.setText("Nieprawidłowa wartość");
+            errorLabel.setVisible(true);
+            hasErrors = true;
+        }
+    }
+
+    public void validatePasswordEquality(String pass1, String pass2, Text errorLabel) {
+        try {
+            if (!pass1.equals(pass2)) {
+                errorLabel.setText("Hasła nie zgadzaja sie");
+                errorLabel.setVisible(true);
+                hasErrors = true;
+            }
+        } catch (NullPointerException ex) {
```

Comment:
Nie powinno się łapać NullPointera. W tym przypadku występuje gdy próbujemy wykonać equals na nullu. Powinno być to w jednym ifie - **if (pass1 == null || pass2 == null || !pass1.equals(pass2))** [reszta]. Przy takim ifie nigdy nie wystąpi NullPointer ponieważ jeśli któryś z haseł jest nullem to nie dojdzie do sprawdzenia trzeciego warunku